.github/renovate.json

```
@@ -22,33 +40,70 @@
         "^eslint$"
       ]
     },
+    {
+      "enabled": true,
+      "groupName": "GraphQL",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": ["^@graphql/", "^graphql$", "^graphql-"]
```

```suggestion
      "matchPackagePatterns": ["^@graphql/", "^graphql$"]
```

It's redundant, right? @TimDiekmann